share/server/dreyfus.js

```diff
@@ -20,7 +20,7 @@ var Dreyfus = (function() {
     } else if (err[0] == "fatal") {
       throw(err);
     }
-    var message = "function raised exception " + err.toSource();
+    var message = "function raised exception " + (err.toSource ? err.toSource() : err.toString());
```

Comment:
Saw a few other cases use `err.stack`. It might not make much of a difference here.

Reply:
I tried `err.stack` and it is undefined. But then `err.toString()` returns `[object Object]` which isn't great either.